2024-01-26 15:04:47 ERROR Elements not present after 0.5 seconds on page: https://capital.com/
2024-01-26 15:04:47 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 293, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-cIj6ROMf-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x558215729f83 <unknown>
#1 0x5582153e2cf7 <unknown>
#2 0x55821543299e <unknown>
#3 0x558215432aa1 <unknown>
#4 0x55821547dd64 <unknown>
#5 0x55821545c0dd <unknown>
#6 0x55821547b006 <unknown>
#7 0x55821545be53 <unknown>
#8 0x558215423dd4 <unknown>
#9 0x5582154251de <unknown>
#10 0x5582156ee531 <unknown>
#11 0x5582156f2455 <unknown>
#12 0x5582156daf55 <unknown>
#13 0x5582156f30ef <unknown>
#14 0x5582156be99f <unknown>
#15 0x558215717008 <unknown>
#16 0x5582157171d7 <unknown>
#17 0x558215729124 <unknown>
#18 0x7fb0a5c94ac3 <unknown>